.github/scripts/run-presto-kerberos-integration-test.sh

```diff
@@ -3,6 +3,7 @@
 set -eo pipefail
 
 # Need Java commands on $PATH, which apparently is not yet the case
+export JAVA_HOME="/usr/lib/jvm/msopenjdk-current"
```

Comment:
not sure what changed in our Dockerfile but somehow the JAVA_HOME changed.
This CI run just fine 5 days ago and our dockerfile hasn't been updated for over a month now.

![Screen Shot 2022-09-27 at 16 29 02](https://user-images.githubusercontent.com/25661381/192489732-4cf5ded2-c812-4745-8b24-d1507dc5ee9a.png)

Reply:
For the record the default `JAVA_HOME=/usr/local/sdkman/candidates/java/current`

Reply:
This feels a bit wonky to me. The environment for this is setup in `.github/workflows/presto-kerberos-integration-test.yml`. If possible we should ensure the environment is correctly setup there where we declare the env, especially if its a very specific location for the jdk.

Reply:
I tried to look into it but that workflow use `mba` which is a CLI that I just know of.
I asked Rai(who built `mba`) and he said had no idea what happened.

 Can we do this in a follow up?